WordPress/Classes/Extensions/UIView+Borders.swift

```diff
@@ -0,0 +1,32 @@
+extension UIView {
```

Comment:
Regarding the top & bottom border styling, will these borders be applied to the cells on the other screens?

Reply:
Not really. But this seems like a good opportunity to generalise it, as there are other places where there are some borders added.

Reply:
Right. I was thinking about _Segments_ & _Verticals_ - will we not need to apply top & bottom borders to in the same fashion?